static/api.jsx

```diff
@@ -1,45 +1,43 @@
-const jQuery = require('jquery');
+import * as qs from 'query-string';
```

Comment:
I usually use `querystringify` because it's quite a bit smaller, but a couple extra kb certainly isn't killing us here

Reply:
yeah, just kept consistent with what we were using in sentry